src/assets/custom/css/_sass/_get-in-touch.scss

```diff
@@ -27,6 +27,7 @@
   @include medium-large-and-extra-large {
     margin-top: #{(-1.5 * $vertical-overlap)};
     padding-top: 60px;
+    @include style-card-box-shadow;
```

Comment:
Just a style thing here, but to keep consistent on layout I would put this as the first style